docker/pulsar-all/pom.xml

```diff
@@ -69,14 +69,6 @@
     <profile>
       <id>docker</id>
       <!-- include the docker image only when docker profile is active -->
-      <dependencies>
```

Comment:
I think we need to keep this due to the error, https://github.com/apache/pulsar/pull/14005/checks#step:11:225